docker/startup.sh

```diff
@@ -19,8 +19,9 @@ declare -a COMMAND=("$@")
 # Consume all arguments before sourcing the environment script
 shift $#
 
-if [ -n "$PYREX_OEROOT" ] && [ -n "$PYREX_INIT_COMMAND" ]; then
-    pushd "$PYREX_OEROOT" > /dev/null
+PYREX_INIT_DIR="${PYREX_INIT_DIR:-$PYREX_OEROOT}"
```

Comment:
I don't think there is any reason for this fallback. Pyrex will always provide the PYREX_INIT_DIR variable when invoking the container, so this can be removed.